apps/webapp/src/components/staking/validators-table.tsx

```diff
@@ -1,72 +1,68 @@
+   */
+  renderStakingActions: (validatorInfo: ValidatorInfo) => ReactNode;
+}
 
+export const ValidatorsTable = ({
+  loading,
+  error,
+  validatorInfos,
+  votingPowerByValidatorInfo,
+  renderStakingActions,
+}: ValidatorsTableProps) => {
   const showError = !!error;
   const showLoading = loading && !validatorInfos.length;
   const showValidators = !showError && !showLoading;
 
   return (
-    <Card>
-      <CardHeader>
-        <CardTitle>Active validators</CardTitle>
-      </CardHeader>
-      <CardContent>
-        <Table className='w-full'>
-          <TableHeader>
-            <TableRow>
-              {HEADERS.map(header => (
-                <TableHead key={header}>{header}</TableHead>
-              ))}
-            </TableRow>
-          </TableHeader>
-          <TableBody>
-            {showError && (
-              <TableRow>
-                <TableCell colSpan={HEADERS.length}>
-                  There was an error loading validators. Please reload the page.
-                </TableCell>
-              </TableRow>
-            )}
+    <Table className='w-full'>
+      <TableHeader>
+        <TableRow>
+          {HEADERS.map(header => (
+            <TableHead key={header}>{header}</TableHead>
+          ))}
+        </TableRow>
+      </TableHeader>
+      <TableBody>
+        {showError && (
+          <TableRow>
+            <TableCell colSpan={HEADERS.length}>
+              There was an error loading validators. Please reload the page.
+            </TableCell>
+          </TableRow>
+        )}
 
-            {showLoading && (
-              <TableRow>
-                <TableCell colSpan={HEADERS.length} className='flex gap-4'>
-                  <Oval width={16} height={16} color='white' secondaryColor='white' />
-                </TableCell>
-              </TableRow>
-            )}
+        {showLoading && (
+          <TableRow>
+            <TableCell colSpan={HEADERS.length} className='flex gap-4'>
+              <Oval width={16} height={16} color='white' secondaryColor='white' />
+            </TableCell>
+          </TableRow>
+        )}
 
-            {showValidators &&
-              validatorInfos.map(validatorInfo => (
-                <ValidatorInfoRow
-                  key={getValidator(validatorInfo).name}
-                  loading={loading}
-                  validatorInfo={validatorInfo}
-                  votingPowerByValidatorInfo={votingPowerByValidatorInfo}
-                />
-              ))}
-          </TableBody>
-        </Table>
-      </CardContent>
-    </Card>
+        {showValidators &&
+          validatorInfos.map(validatorInfo => (
+            <ValidatorInfoRow
+              key={getValidator(validatorInfo).name}
+              loading={loading}
+              validatorInfo={validatorInfo}
+              votingPowerByValidatorInfo={votingPowerByValidatorInfo}
+              staking={renderStakingActions(validatorInfo)}
+            />
```

Comment:
We should work towards the design in https://github.com/penumbra-zone/web/pull/569#issuecomment-1957738474 , this should be a component for rendering a delegation token, and it should be the same component as is used for the user's delegations.